services/automerge/automerge.go

```diff
@@ -62,16 +65,6 @@ func addToQueue(pr *issues_model.PullRequest, sha string) {
 // ScheduleAutoMerge if schedule is false and no error, pull can be merged directly
 func ScheduleAutoMerge(ctx context.Context, doer *user_model.User, pull *issues_model.PullRequest, style repo_model.MergeStyle, message string) (scheduled bool, err error) {
 	err = db.WithTx(ctx, func(ctx context.Context) error {
-		lastCommitStatus, err := pull_service.GetPullRequestCommitStatusState(ctx, pull)
```

Comment:
The check is not enough, we at least call `pull_service.CheckPullMergable` to check whether the pull request is mergeable. If it returns false, we need to return a message to the UI.

Reply:
We should allow to start checking auto merge even if commit statuses not PASS.